src/main/resources/assets/pipez/lang/de_de.json

```diff
@@ -1,45 +1,45 @@
-  "block.pipez.fluid_pipe": "Flüssigkeits Röhre",
-  "block.pipez.energy_pipe": "Energie Röhre",
-  "block.pipez.universal_pipe": "Universele Röhre",
-  "block.pipez.gas_pipe": "Gas Röhre",
-  "item.pipez.basic_upgrade": "Normale Röhre Verbesserung",
-  "item.pipez.improved_upgrade": "Verbesserte Röhre Verbesserung",
-  "item.pipez.advanced_upgrade": "Erweiterte Röhre Verbesserung",
-  "item.pipez.ultimate_upgrade": "Ultimative Röhre Verbesserung",
-  "item.pipez.infinity_upgrade": "Unendliche Röhre Verbesserung",
+  "itemGroup.pipez": "Pipez",
+  "block.pipez.item_pipe": "Item Rohr",
+  "block.pipez.fluid_pipe": "Flüssigkeits Rohr",
+  "block.pipez.energy_pipe": "Strom Rohr",
+  "block.pipez.universal_pipe": "Universelle Rohr",
+  "block.pipez.gas_pipe": "Gas Rohr",
+  "item.pipez.basic_upgrade": "Normale Rohr Verbesserung",
+  "item.pipez.improved_upgrade": "Verbesserte Rohr Verbesserung",
+  "item.pipez.advanced_upgrade": "Fortgeschrittene Rohr Verbesserung",
+  "item.pipez.ultimate_upgrade": "Ultimative Rohr Verbesserung",
+  "item.pipez.infinity_upgrade": "Unendliche Rohr Verbesserung",
   "item.pipez.wrench": "Röhren Zange",
   "item.pipez.filter_destination_tool": "Filter Ziel Gerät",
-  "tooltip.pipez.wrench": "Klicke auf eine Röhre um sie einzustellen",
+  "tooltip.pipez.wrench": "Klicke auf ein Rohr, um sie einzustellen",
   "tooltip.pipez.distribution": "Verteilung: %s",
   "tooltip.pipez.distribution.nearest": "Nächste zuerst",
   "tooltip.pipez.distribution.furthest": "Weiteste zuert",
   "tooltip.pipez.distribution.round_robin": "Rundlauf",
   "tooltip.pipez.distribution.random": "Zufällig",
   "tooltip.pipez.redstone_mode": "Rotstein Modus: %s",
   "tooltip.pipez.redstone_mode.ignored": "Ignoriert",
-  "tooltip.pipez.redstone_mode.off_when_powered": "Aus Wenn Aktiviert",
-  "tooltip.pipez.redstone_mode.on_when_powered": "An Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.off_when_powered": "Aus wenn Redstone aktiviert",
+  "tooltip.pipez.redstone_mode.on_when_powered": "An wenn Aktiviert",
   "tooltip.pipez.redstone_mode.always_off": "Immer Aus",
   "tooltip.pipez.filter_mode": "Filter Modus: %s",
   "tooltip.pipez.filter_mode.whitelist": "Erlaubt",
   "tooltip.pipez.filter_mode.blacklist": "Nicht Erlaubt",
   "tooltip.pipez.filter.not": "Nicht",
   "tooltip.pipez.filter.accepts_tag": "Alle %s",
-  "tooltip.pipez.filter.item_tag.description": "Tags Präfix mit # oder item IDs",
-  "tooltip.pipez.filter.nbt_string.description": "NBT Data",
-  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT tags haben",
-  "tooltip.pipez.filter.nbt.exact": "Genaue übereinstimmung NBT data",
+  "tooltip.pipez.filter.item_tag.description": "Präfix Stichworte mit # oder Item IDs",
+  "tooltip.pipez.filter.nbt_string.description": "NBT Daten",
+  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT Stichworte haben",
+  "tooltip.pipez.filter.nbt.exact": "Genaue Übereinstimmung der Stichwort Daten",
```

Comment:
"Stichwort"??

Reply:
was the straight translation. Should i leave "tag"?

Reply:
Yeah

Reply:
done